source/op/prod_env_mat_multi_device.cc

```diff
@@ -147,6 +147,27 @@ REGISTER_OP("DescrptSeR")
     .Output("rij: T")
     .Output("nlist: int32"); 
 
+REGISTER_OP("ProdEnvMatAMix")
+    .Attr("T: {float, double} = DT_DOUBLE")
+    .Input("coord: T")          //atomic coordinates
+    .Input("type: int32")       //atomic type
+    .Input("natoms: int32")     //local atomic number; each type atomic number
+    .Input("box : T")
+    .Input("mesh : int32")
+    .Input("davg: T")           //average value of data
+    .Input("dstd: T")           //standard deviation
+    .Attr("rcut_a: float")      //no use
+    .Attr("rcut_r: float")
+    .Attr("rcut_r_smth: float")
+    .Attr("sel_a: list(int)")
+    .Attr("sel_r: list(int)")   //all zero
+    .Output("descrpt: T")
+    .Output("descrpt_deriv: T")
+    .Output("rij: T")
+    .Output("nlist: int32")
+    .Output("ntype: int32")
+    .Output("nmask: T");;
```

Comment:
Maybe you could add a summary to the new OP, especially the difference from the old OP.

```cpp
.Doc(R"(Something

something something
something.)");
```